apps/comps/app/competitions/[id]/page.tsx

```diff
@@ -89,16 +90,17 @@ export default function CompetitionPage({
   }
 
   return (
-    <>
+    <div style={{ marginTop: "-40px" }}>
```

Comment:
The padding and margin rules that exist currently make it a little tricky to get the banner to be a child component and still push up against the nav, as per the design, this negative margin combined with a matching positive margin for the breadcrums work afaict.